### ISSUE TITLE
Jupyter-like focus indicator should be sticky

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#125338]
The focus indicator should not be removed if the focus moves insider the cell editor or into an output. The way it is right now, I'm still prone to getting lost particularly when switching back and forth between editors.

![ReckEYiOnL](https://user-images.githubusercontent.com/4674940/120550310-8c8ef300-c3a9-11eb-83a1-fdaa497fb334.gif)
